### DISCUSSION TITLE
Ignore Zero values in XY line Chart

### DISCUSSION THREAD

**Discussion by donwynne** [jfree/jfreechart#289] [General]
Hiya,

I have a XY Line chart wherein i doesnt want to show the value zero eventhough the graph is plotted at the below i.e just ignorezerovalues as in stackedbar chart which works pretty good.

Any suggestions ?

**Comment by trashgod** [comment]
What happens when you set the [`autoRangeIncludesZero`](https://www.jfree.org/jfreechart/javadoc/org/jfree/chart/axis/NumberAxis.html) flag on the desired axis?
